test/unit/services/finance/charging_service_test.rb

```diff
@@ -98,7 +98,8 @@ def build_payment_gateway(type)
       end
 
       def payment_gateway_options
-        { login: 'sk_test_4eC39HqLyjWDarjtT1zdp7dc', publishable_key: 'pk_test_TYooMQauvdEDq54NiTphI7jx', endpoint_secret: 'some-secret' }
+        { login: 'sk_test_4eC39HqLyjWDarjtT1zdp7dc', publishable_key: 'pk_test_TYooMQauvdEDq54NiTphI7jx',
+          endpoint_secret: 'some-secret', description: Finance::StripeChargeService::PAYMENT_DESCRIPTION }
```

Comment:
should this be the default already `PAYMENT_DESCRIPTION`?

Reply:
You're right, actually, this description just needs to get added to the "expected" params